pkg/rancher-desktop/config/__tests__/settings.spec.ts

```diff
@@ -79,6 +79,7 @@ describe('settings', () => {
   });
 
   const fullDefaults = {
+    version:     10,
```

Comment:
This is going to bite us when we bump the version, isn't it. Ah well.

Reply:
Good point - it will.